tests/unit/confluence/test_labels.py

```diff
@@ -0,0 +1,151 @@
+"""Unit tests for the LabelsMixin class."""
+
+from unittest.mock import patch
+
+import pytest
+import requests
+
+from mcp_atlassian.confluence.labels import LabelsMixin
+from mcp_atlassian.models.confluence import ConfluenceLabel
+
+
+class TestLabelssMixin:
+    """Tests for the LabelsMixin class."""
+
+    @pytest.fixture
+    def labels_mixin(self, confluence_client):
+        """Create a LabelsMixin instance for testing."""
+        # LabelsMixin inherits from ConfluenceClient, so we need to create it properly
+        with patch(
+            "mcp_atlassian.confluence.labels.ConfluenceClient.__init__"
+        ) as mock_init:
+            mock_init.return_value = None
+            mixin = LabelsMixin()
+            # Copy the necessary attributes from our mocked client
+            mixin.confluence = confluence_client.confluence
+            mixin.config = confluence_client.config
+            mixin.preprocessor = confluence_client.preprocessor
+            return mixin
+
+    def test_get_page_labels_success(self, labels_mixin):
+        """Test get_page_labels with success response."""
+        # Setup
+        page_id = "12345"
+        # Configure the mock to return a successful response
+        labels_mixin.confluence.get_page_labels.return_value = {
+            "results": [
+                {
+                    "id": "456789123",
+                    "prefix": "global",
+                    "name": "meeting-notes",
+                    "label": "meeting-notes",
+                },
+                {
+                    "id": "456789124",
+                    "prefix": "my",
+                    "name": "important",
+                },
+                {
+                    "id": "456789125",
+                    "name": "test",
+                },
+            ]
+        }
```

Comment:
The return value is already set by the labels_mixin fixture using `MOCK_LABELS_RESPONSE`. No need to set it again here.


```suggestion
```